packages/js/src/generators/library/library.ts

```diff
@@ -200,6 +200,13 @@ function addProject(
         command: `node ${publishScriptPath} ${options.name} {args.ver} {args.tag}`,
         dependsOn: ['build'],
       };
+
+      projectConfiguration.targets['local-registry'] = {
```

Comment:
Should this go on the project, wherever it is, or should it specifically go on the root project? I'd imagine that you wouldn't want multiple of this target to run if you ran e2e in parallel, and e2e would likely depend on this right?

Reply:
added a check for root project, so the local-registry target will only be added to the publishable root project.